docs/README.md

```diff
@@ -0,0 +1,139 @@
+
+If you're not aware yet, then:
+
+#### **WARNING**: HERE BE DRAGONS! THIS IS A FRAGILE PROJECT
+
+# Getting Started
+
+To get started, we suggest these methods. They are ordered from most suggested to least suggested.
+In all of these methods, you'll need your Doom Emacs configuration. It should contain the following three files: 
+`config.el`, `init.el` and `packages.el`. If you don't already have an existing `doom-emacs` configuration, you can use the contents of `test/doom.d` as a template.
+
+The Doom configuration will be referred to as `./doom.d` in these snippets. You can name it whatever you like.
+
+## Flake + Home-Manager
+
+`File: flake.nix`
+```nix
+{
+  inputs = {
+    home-manager.url = "github:nix-community/home-manager";
+    nix-doom-emacs.url = "github:nix-community/nix-doom-emacs";
+  };
+
+  outputs = {
+    self,
+    nixpkgs,
+    lib,
+    home-manager,
+    nix-doom-emacs,
+    ...
+  }:
+    let
+      system      = "x86_64-linux";
+      specialArgs = { inherit inputs; };
+    in {
+    nixosConfigurations.exampleHost = lib.nixosSystem {
+      inherit system specialArgs;
+      modules = [
+        ./default.nix
+        home-manager.nixosModules.home-manager
+        {
+          home-manager.users.exampleUser.imports = [ ./home.nix ];
+        }
+      ];
+    };
+  };
+}
+
+```
+
+`File: home.nix`
+```nix
+{ config, pkgs, inputs, ... }: {
+  imports = [ inputs.nix-doom-emacs.hmModule ];
+
+  # ...
+  programs.doom-emacs = {
+    enable = true;
+    doomPrivateDir = ./doom.d; # Directory containing your config.el, init.el
+                               # and packages.el files
+  };
+  # ...
+}
+```
+
+## Non-Flake Home-Manager
+
+```nix
+{ pkgs, ... }:
+
+let
+  doom-emacs = pkgs.callPackage (builtins.fetchTarball {
+    url = https://github.com/nix-community/nix-doom-emacs/archive/master.tar.gz;
+  }) {
+    doomPrivateDir = ./doom.d;  # Directory containing your config.el, init.el
+                                # and packages.el files
+  };
+in {
+  home.packages = [ doom-emacs ];
+}
+```
+
+
+## Standalone
+
+Using Nix-Doom-Emacs standalone isn't recommended, especially if you're a beginner.
+
+`File: flake.nix`
+```nix
+{
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+    nix-doom-emacs.url = "github:nix-community/nix-doom-emacs";
+  };
+  
+  outputs = {
+    self,
+    nixpkgs,
+    lib,
+    nix-doom-emacs,
+    ...
+  }:
+    let
+      system      = "x86_64-linux";
+      specialArgs = { inherit inputs; };
+    in {
+    nixosConfigurations.exampleHost = lib.nixosSystem {
+      inherit system specialArgs;
+      modules = [
+        ./default.nix
+        # ...
+      ];
+    };
+  };
+}
+```
+
+`File: default.nix`
+```nix
+{ config, nixpkgs, lib, inputs }: {
+  # ...
+  environment.systemPackages = 
+    let
+      doom-emacs = inputs.nix-doom-emacs.packages.${system}.default.override {
```

Comment:
Can you add a comment here explaining that you can see all supported parameters inside our `default.nix` file?